src/avram/model.cr

```diff
@@ -227,7 +237,9 @@ abstract class Avram::Model
     {% else %}
       {% value = nil %}
     {% end %}
-    {% COLUMNS << {name: type_declaration.var, type: data_type, nilable: nilable.id, autogenerated: autogenerated, value: value} %}
+    {% serialized = is_serialized || data_type == "JSON::Any".id %}
```

Comment:
You might experiment, but I think if you brought over the additional changes I made in the `crystal-pg` PR I think you might be able to remove the `JSON::Any` check since it would just work as before

Reply:
Yup! I just used your branch directly, and it looks like it all still works.